cmd/bootstrap/cmd/key_test.go

```diff
@@ -19,8 +19,12 @@ var happyPathRegex = `^will generate networking key` +
 	`generated networking key` +
 	`will generate staking key` +
 	`generated staking key` +
+	`will generate machine account key` +
+	`generated machine account key` +
 	`assembling node information` +
 	`encoded public staking and network keys` +
+	`assembling machine account information` +
+	`encoded public machine account key` +
```

Comment:
Can we check that the `node-info` and `machine-account-key` files were written as well? As in line 44-46.